packages/good-design/src/stories/apps/goodid/GoodId.stories.tsx

```diff
@@ -76,17 +76,19 @@ export const SegmentationScreen = {
 };
 
 export const SegmentationFlow = {
-  render: () => (
+  render: (args: any) => (
     <W3Wrapper withMetaMask={true} env="staging">
-      <VStack>
+      <VStack {...args}>
         <Text variant="browse-wrap" fontSize="sm">
           For testing purposes. this flow is using staging/QA contracts
         </Text>
         <SegmentationController />
       </VStack>
     </W3Wrapper>
   ),
-  args: {}
+  args: {
+    width: 375
```

Comment:
still fixed width

Reply:
Yes, there is initial value. and you can now configure it by the controls in storybook.
Thats the whole idea. applies to all other comments